docs/src/main/sphinx/release/release-432.md

```diff
@@ -0,0 +1,57 @@
+# Release 432 (2 Nov 2023)
+
+:::{note}
+From this point forward, release notes will include a ⚠️ symbol to designate
```

Comment:
Release notes now include a ⚠️ symbol to highlight any changes as potentially breaking changes. The following changes are considered and may require adjustments: